### DISCUSSION TITLE
Hi new translators!

### DISCUSSION THREAD

**Discussion by tesseralis** [reactjs/reactjs.org-translation#233] [Team Posts]
@reactjs/reactjs-org-localization Hi everyone! Please use this space to discuss anything translation related or to ask questions to your other translators!

Please start by looking at the generated progress issue: (e.g. https://github.com/reactjs/id.reactjs.org/issues/1) and start contributing!

**Comment by nampdn** [comment]
Can I have a definition of the term “event delegation” in home/an-application?

**Comment by tesseralis** [comment]
If you search it on google you get a couple of good results! For example: https://learn.jquery.com/events/event-delegation/

**Comment by hg-pyun** [comment]
Thank you for guide :)

**Comment by nampdn** [comment]
Can I translate text placed in a source code component or only content folder?

**Comment by tesseralis** [comment]
You can translate text in source code! One of the advantages of this method is that you can translate it directly without needing a localization library :D

For an example, see the homepage of the simplified chinese translation:

https://github.com/reactjs/zh-hans.reactjs.org/blob/master/src/pages/index.js
